ansible/templates/centos7/deps.sh.j2

```diff
@@ -10,14 +10,21 @@ yum -y install epel-release
 
 yum -y install git
 
-yum -y install gcc gcc-c++ zlib-devel
+yum -y install python-pip
 
+{% if system_site_packages %}
+yum -y install \
+    python-pillow \
+    numpy
```

Comment:
Do we want to add ``python-requests``?
similar to https://github.com/openmicroscopy/ansible-role-omero-web-runtime/pull/3/files